setup_jenkins.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+sed 's/DOCKER_REGISTRY=.*/DOCKER_REGISTRY=ghcr.io\/candig/; s/WORKING_DIR=.*//' etc/env/example.env > .env
```

Comment:
Better to add this as instructions for user to modify in documentation for example.env.

Reply:
Are we permanently switching from Dockerhub to Github Container Registry? If so, we can just change the value directly in example.env.